tests/manual/spellchecking.md

```diff
@@ -0,0 +1,6 @@
+## Input (typing) feature with spell checking
+
+Try to correct all misspelled words using native spell checking mechanism in the browser.
+
+* Words should be corrected and selection placed after corrected word. _In Safari selection is placed
+at the beginning of corrected word.
```

Comment:
Also, if this is a bug, you can link to the issue on GH to explain that this is a bug.